### DISCUSSION TITLE
Optional chaining in query parameters for rest connectors

### DISCUSSION THREAD

**Discussion by nehalicious** [hasura/graphql-engine#9130] [General]
https://hasura.io/docs/latest/api-reference/kriti-templating/#kriti-optional-chaining

Hey, I'm trying to configure a rest connector to use optional chaining for query parameters. Essentially something like this:
```foo: '{{$body.input?.foo}}'```

However, when i try to save this, I see an error saying 
```"LexError: Invalid Lexeme: '?}}\"'"```

does anyone know what could be going wrong? If I look at the documentation, this seems like it should be fairly straighforward https://hasura.io/docs/latest/api-reference/kriti-templating/#kriti-optional-chaining

**Comment by adas98012** [comment]
Thanks for bringing this to our attention. I am creating a Github [Issue](https://github.com/hasura/graphql-engine/issues/9152) for this.
